CHANGES.md

```diff
@@ -0,0 +1,295 @@
+- Moved two upgrade spawns over to a new shelf prop so they spawn when transitioning from map 1.
+
+##### Blood Harvest 2:
+
+- Removed shadows from the breakwall prop_dynamic entities.
+
+#### Navmesh:
+##### Hard Rain 2:
+
+- Fixed -9999 flow areas.
+- Survivor bots can save you in more places.
+- Lot of bad connections fixed.
+- Lot of merging.
+
+#### Talker:
+
+- Anything for the survivors that uses "then " will now use "then self" instead.
+- Swapped Francis' ArriveBoat02 and ArriveBoat03 placements.
+- Fixed two of Coach's rules using "IsTalkCoach" twice.
+- Many tweaks to rules referring to leaving the starting area.
+- Fixed RemarkC13M1DownHereMechanic rule having the wrong name. (Thanks, Haruko)
+- Swapped Louis' ArriveChopper line placements.
+- Deleted C6M3_ElevatorCancel rules for the L4D2 survivors (they had no real functionality).
+- Added IsNotAlone to the C1M1Elevator lines for Coach, Gambler, Mechanic and Producer.
+- Added NoOneInSafeSpot to the "outside" vehicle nags.
+- Split the vehicle nags for C6M3 for Coach, Gambler and Producer.
+- Added NotAloneOutsideSafeSpot to C5M5RunToHeliRareCoach.
+- Added the FinalVehicleArrived concept for Dark Carnival and Swamp Fever.
+- Updated criteria definitions for IsIncappedStarted and IsIncappedBleeding.
+- Added IsNotAlone to the c1m1_enter_elevator concept for Coach, Gambler, Mechanic and Producer.
+
+#### Models:
+
+- Improved collision and usability of the Forklift hittable for Versus Tanks.
+
+#### Materials:
+
+- Reduced the env map intensity on "models/props/cs_militia/refrigerator01.mdl".
+- Reduced the env map intensity on pipebomb, hunting rifle, and worn variants.
+
+1.1.0 (Sep 24th, 2021)
+======================
+Summary
+-------
+
+- Release candidate
+- 5 contributors
+- 6 pull requests
+
+### What's new
+
+It is exactly 1 year since L4D2's Last Stand Update was released.
+
+This update is a huge re-organization with modest re-writes and a lot more fixes.
+
+Anyone may test these and report issues. Valve will be kept up-to-date and ship when they can.
+
+Note: Customization scripts [#13](https://github.com/Tsuey/L4D2-Community-Update/issues/13) are pending, but bandaids [SirPlease/L4D2-Competitive-Rework#320](https://github.com/SirPlease/L4D2-Competitive-Rework/pull/320) have been applied with competitive config's simple removal of "anv_versus.nut".
+
+#### Lump:
+
+- Lump edits to account for changed weapon density values.
+- Hard Rain 2 and 3: Added a prop to make it easier for infected bots to navigate.
+- Blood Harvest 2: Added missing breakwall texture to wall in warehouse.
+- Blood Harvert 4: Removed unreachable pills spawn.
+
+#### Navmesh:
+
+- Various nav fixes for -9999 flow areas.
+- Improved and fixed bot nav in various maps<!-- including..-->.
+- Harmless file size optimization through merging, deleting and ID compression.
+- Dead Center 2: Fixed nav areas in order for the warp_to_checkpoint concommand to work properly.
```

Comment:
What is "concommand"? And why isn't `warp_to_checkpoint` available in a normal console?

Reply:
It could be corrected to warp_all_survivors_to_checkpoint.

As for concommand, personally I refer to them as cvar, or CVAR, or Convar -- everyone is different, but I guess it wouldn't hurt to pick one for consistency.